partcad/src/partcad/ai_openai.py

```
@@ -85,38 +86,43 @@ def generate_openai(
+            pc_logging.debug("Content: %s", content)
+            if content.is_image:
+                return {
                     "type": "image_url",
                     "image_url": {
                         "url": "data:%s;base64,%s"
                         % (
-                            mimetypes.guess_type(filename, False)[0],
-                            base64.b64encode(Path(filename).read_bytes()).decode(),
+                            mimetypes.guess_type(content.filename, False)[0],
+                            base64.b64encode(Path(content.filename).read_bytes()).decode(),
                         ),
```

Ruff (0.8.2)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)